chart/templates/ingressroute.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.ingressRoute.enabled -}}
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: {{ template "microservice-authentication.fullname" . }}-ingressroute
+spec:
+  entryPoints:
+    - web
+  routes:
+    - match: Host(`auth.anon.dev.ubivius.tk`) && PathPrefix(`/`)
```

Comment:
Maybe use your value you went ahead and put in?

Reply:
Oups...